apps/meteor/server/models/raw/Subscriptions.ts

```diff
@@ -594,6 +605,21 @@ export class SubscriptionsRaw extends BaseRaw<ISubscription> implements ISubscri
 		return this.updateMany(query, update);
 	}
 
+	updateNameAndFnameByVisitorIds(visitorIds: string[], name: string): Promise<UpdateResult | Document> {
+		const query = { 't': 'l' as const, 'v._id': { $in: visitorIds } };
+
+		const update = {
+			$set: {
+				name,
+				lowerCaseName: name.toLowerCase(),
+				fname: name,
+				lowerCaseFName: name.toLowerCase(),
```

Comment:
Any reason for this new property? Didn't get it...

Reply:
(same as in the other comment)